deploy-edgeX-snaps.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Set any of the following snap environment variables. Note you must also set TEST_ENV=localhost
+# (in the env.sh file) as well. Note, EF_SNAP takes precedence over EF_CHANNEL and ASC_SNAP
+# takes precedence over ASC_CHANNEL.
+
+# If EF_CHANNEL is not set, it defaults to 'stable'.
+# Accepted values are: stable | beta | edge | candidate | channel/track e.g. latest/edge
```

Comment:
I would just drop "Accepted values..." altogether.  "channel/track e.g. latest/edge" doesn't really make sense. A channel is a triplet of "track" + "risk" + "branch", so "latest/edge" is really track/risk, not a "channel/track". In the end, someone who knows snaps will understand how to set this.